src/components/addForm/addForm.html

```diff
@@ -0,0 +1,22 @@
+<form>
+  <fieldset>
+    <!--label or legend for description?-->
+    <label for="description">DESCRIPTION</label>
```

Comment:
You can add `text-transform: uppercase` to all the labels in your CSS, so that you don't have to write it in caps here.